ote/src/cljs/ote/app/controller/transport_operator.cljs

```diff
@@ -384,9 +401,14 @@
     (if id
       (do
         (comm/get! (str "t-operator/" id)
-                   {:on-success (send-async! ->EditTransportOperatorResponse)})
-        (assoc app :transport-operator-loaded? false))
-      (assoc app :transport-operator-loaded? true)))
+                   {:on-success (send-async! ->EditTransportOperatorResponse)
+                    :on-failure (send-async! ->ServerError)})
+        (-> app
+            (get-country-list)
+            (assoc :transport-operator-loaded? false)))
+      (-> app
+          (get-country-list)
+          (assoc :transport-operator-loaded? true))))
```

Comment:
Implementaition is ok, but please consider also following alternative
```
EditTransportOperator
(process-event [{id :id} app]
               (->
                 (if id
                   (do
                     (comm/get! (str "t-operator/" id)
                                {:on-success (send-async! ->EditTransportOperatorResponse)
                                 :on-failure (send-async! ->ServerError)})
                     (assoc app :transport-operator-loaded? false))
                   (assoc app :transport-operator-loaded? true))
                 (get-country-list)))
```
(process-event [{id :id} app]
               (->
                 (if id
                   (do
                     (comm/get! (str "t-operator/" id)
                                {:on-success (send-async! ->EditTransportOperatorResponse)
                                 :on-failure (send-async! ->ServerError)})
                     (assoc app :transport-operator-loaded? false))
                   (assoc app :transport-operator-loaded? true))
                 (get-country-list)))